Resources/Prototypes/Entities/Objects/Misc/bones.yml

```diff
@@ -0,0 +1,41 @@
+- type: entity
+  name: animal bone
+  parent: BaseItem
+  id: Bone
+  description: It probably used to be inside something.
```

Comment:
Step 1: be skeleton
Step 2: be made of bones
Step 3: die
Step 4: don't drop bones